### DISCUSSION TITLE
Kubernetes deploy

### DISCUSSION THREAD

**Discussion by marekvesely-direct** [canarytrace/documentation#64] [Synthetic] [answered]
Dobry den,

v manifestu pro Kubernetes se zminujete o pripojeni na github, k cemu tam slouzi, je potreba ho nastavovat? ENV promenna SPEC se nastavuje js soubor, jak ho definovat, je to lokace na serveru nebo jak ho nastavit?

Dekuji,

Vesely

**Comment by rdpanek** [comment, marked as answer]
🇨🇿 [ czech ]

Dobry den @marekvesely-direct , dekuji za dotaz.

Pokud se pouziva Canarytrace Smoke, github a ani zadny jiny repo neni potreba, zadavaji se pouze cilove adresy do `BASE_URL`
Pokud se smoke edice nepouziva, tak Canarytrace potrebuje monitor script / test case, ktery ma provest. Takze je potreba zadat nejake repo ve kterem jsou testy. Pri spusteni dojde k naklonovani repozitare, checkout na konkretni revizi a pusteni souboru ve kterem je test case.

Diky Dan
